.github/workflows/push.yml

```diff
@@ -17,6 +17,9 @@ on:
   pull_request:
     branches:
       - '**'
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
```

Comment:
I believe `github.head_ref` might be unsafe, because to my understanding it does not include the source organization / user, thus the default branch names such as `patch-1` might collide.

Can you try `github.event.pull_request.url || github.run_id`?

Reply:
I'm not sure, but looking at the document I think `ref_name` would be the feature branch name, and `head_ref` would be fully qualified like `ref`. But I will verify. Note that this code is copied [directly from the documentation](https://docs.github.com/en/actions/using-jobs/using-concurrency#example-using-a-fallback-value).